ADAL/src/cache/ADAuthenticationContext+TokenCaching.h

```diff
@@ -31,11 +31,13 @@
 //the item to be stored.
 - (void)updateCacheToResult:(ADAuthenticationResult*)result
                   cacheItem:(ADTokenCacheItem*)cacheItem
-           withRefreshToken:(NSString*)refreshToken;
+           withRefreshToken:(NSString*)refreshToken
+       requestCorrelationId:(NSUUID*)requestCorrelationId;
```

Comment:
Hrm, I would have thought we'd already have the token cache item with the refresh token laying around somewhere. It's possible it's just not factored well enough. I'll flag this code path for some cleaning up in the future.